cmd/gendoc/main.go

```diff
@@ -0,0 +1,245 @@
+package main
```

Comment:
Why do we need gendoc when we have goland godoc?

I don't see much value in exporting to MD when we already have HTML.

Reply:
The generator also generated Examples, and output data size. And there are some more ideas how to make whole effort easier with the specific generator.

Reply:
Ok